modules/virt-accessing-vm-yaml-ssh.adoc

```diff
@@ -0,0 +1,135 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-accessing-vm-consoles.adoc
+
+[id="virt-accessing-vm-yaml-ssh_{context}"]
+= Accessing a virtual machine via SSH with YAML configurations
+
+You can enable a SSH connection to a virtual machine (VM) without the need to run the `virtctl expose` command. When the YAML file for the VM and the YAML file for the service are configured and applied, the service forwards the SSH traffic to the VM.
+
+The following examples show the configurations for the VM's YAML file and the service YAML file.
+
+.Prerequisites
+* Install the OpenShift CLI (`oc`).
+
```

Comment:
Please add a prerequisite:
Create a new namespace:
`oc create namespace ssh-ns` (or any other name you see fit).